interface/modules/custom_modules/oe-module-comlink-telehealth/src/TelehealthGlobalConfig.php

```diff
@@ -0,0 +1,182 @@
+        return $this->cryptoGen->decryptStandard($encryptedValue);
+    }
+
+    public function getRegistrationAPICmsId()
+    {
+        return $this->getGlobalSetting(self::COMLINK_VIDEO_TELEHEALTH_CMS_ID);
+    }
+
+    public function shouldAutoProvisionProviders(): bool
+    {
+        $setting = $this->getGlobalSetting(self::COMLINK_AUTO_PROVISION_PROVIDER);
+        return $setting !== "";
+    }
+
+    public function getGlobalSetting($settingKey)
+    {
+        global $GLOBALS;
+        // don't like this as php 8.1 requires this but OpenEMR works with globals and this is annoying.
+        return $GLOBALS[$settingKey] ?? null;
+    }
+
+    public function getAppRegistrationCodeLength()
+    {
+        return self::APP_REGISTRATION_CODE_LENGTH;
+    }
+
+    public function getGlobalSettingSectionConfiguration()
+    {
+        $settings = [
+            self::COMLINK_VIDEO_REGISTRATION_API => [
+                'title' => 'Telehealth Registration URI'
+                ,'description' => 'Registration endpoint URI'
+                ,'type' => GlobalSetting::DATA_TYPE_TEXT
+                ,'default' => ''
+            ]
+            ,self::COMLINK_VIDEO_TELEHEALTH_API => [
+                'title' => 'Telehealth Video API URI'
+                ,'description' => 'The URI for the video bridge api'
+                ,'type' => GlobalSetting::DATA_TYPE_TEXT
+                ,'default' => ''
+            ]
+            ,self::COMLINK_VIDEO_API_USER_ID => [
+                'title' => 'Telehealth Installation User ID'
+                ,'description' => 'This is your unique video application api user id.  Contact ComLink if you have not received it'
+                ,'type' => GlobalSetting::DATA_TYPE_TEXT
+                ,'default' => ''
+            ]
+            ,self::COMLINK_VIDEO_API_USER_PASSWORD => [
+                'title' => 'Telehealth Installation User Password (Encrypted)'
+                ,'description' => 'This is your unique video application api password.  Contact ComLink if you have not received it'
+                ,'type' => GlobalSetting::DATA_TYPE_ENCRYPTED
+                ,'default' => ''
+            ]
+            ,self::COMLINK_VIDEO_TELEHEALTH_CMS_ID => [
+                'title' => 'Telehealth Installation CMSID'
+                ,'description' => 'This is your unique video application CMSID.  Contact ComLink if you have not received it'
+                ,'type' => GlobalSetting::DATA_TYPE_TEXT
+                ,'default' => ''
+            ]
+            ,self::COMLINK_VIDEO_TELEHEALTH_CMS_ID => [
+                'title' => 'Telehealth Installation CMSID'
+                ,'description' => 'This is your unique video application CMSID.  Contact ComLink if you have not received it'
+                ,'type' => GlobalSetting::DATA_TYPE_TEXT
+                ,'default' => ''
+            ]
+            ,self::COMLINK_AUTO_PROVISION_PROVIDER => [
+                'title' => 'Auto Register Providers For  Telehealth'
```

Comment:
remove the double spacing in this string

Reply:
fixed